code/src/components/Footer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+export const Footer = () => {
+  return (
+    <FooterDiv>
+    <div>
+      <p>This project is made by Gabriella Bolin @Technigo Bootcamp 2020</p>
+    </div>
+    <div>Icons made by  
+      <a href="https://www.flaticon.com/authors/freepik" title="Freepik"> Freepik </a> 
+      from 
+      <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com </a>
+    </div>
+    </FooterDiv>
+  );
+};
+
+const FooterDiv = styled.div`
```

Comment:
Sorry, but... why not a footer 😉

Reply:
Since the component exported is namned that; `export const Footer = ()` then I cannot make another const named Footer. But changed the name to FooterWrapper at least :)

Reply:
Sorry, I meant the element `styled.footer``` 😄